index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="wrapper">
+        <nav class="header">
+            <a class="logo" href="#">My Logo</a>
+            <img src="images/Union.png" alt="">
+        </nav>
+        <section>
+
+            <div class="headline">
+
+                <div class="headline-heading">
+                    <h1>I’m Ahmed Glad to see you!</h1>
```

Comment:
- [Optional] Please use an indentation of two spaces in all your documents for readability. This is a convention and also [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

- Please in all future projects, do not create a new PR to request a re-review. You should always implement changes and submit the same pull request for a re-review.